src/java.base/share/classes/jdk/internal/module/SystemModuleFinders.java

```diff
@@ -378,8 +362,6 @@ public byte[] generate(String algorithm) {
 
     /**
      * Holder class for the ImageReader
```

Comment:
In passing, you would mind adding a full stop after ImageReader. Only noticed it with the removal of the apinote.

Reply:
Added a full stop at the end of the SystemImage class description.